packages/apps/types/src/schema/thread.ts

```diff
@@ -42,7 +42,7 @@ export class MessageType extends TypedObject({ typename: 'dxos.org/type/Message'
 export class ThreadType extends TypedObject({ typename: 'dxos.org/type/Thread', version: '0.1.0' })({
   name: S.optional(S.String),
   anchor: S.optional(S.String),
-  resolved: S.optional(S.Boolean),
+  status: S.optional(S.Union(S.Literal('staged'), S.Literal('active'), S.Literal('resolved'))),
```

Comment:
is it possible to have a persisted thread with status of `staged`?

Reply:
@wittjosiah Not currently, but we create the in memory thread with the same schema.

Reply:
wondering if it would be possible to omit that field from the in memory ones? not sure if it's worth the trouble but feels weird to me to have `staged` be a valid value for the db schema

Reply:
Any thoughts on how we might achieve that?